demisto_sdk/common/tools.py

```diff
@@ -134,6 +134,28 @@ def filter_packagify_changes(modified_files, added_files, removed_files, tag='ma
     return modified_files, updated_added_files, removed_files
 
 
+def get_child_directories(directory):
```

Comment:
isn't there something built in for this?

Reply:
you tell me. is there?